### PR TITLE
Add windows setup guide

### LINKED ISSUE #115: Add documentation on how to get developing on Windows through WSL 2
Although parabeac-core development on Windows is possible, it currently requires quite some setup.
A documented guide on how to set this up with the help of [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/about) would help others to get starting.

### PR DESCRIPTION
**Pull request by roel-de-vries**
Related issue: #115 

This PR adds a comprehensive guide on how to get starting with Parabeac-core on Windows through WSL 2.
Any feedback about content and comments regarding grammar is welcome!

### CODE REVIEW

**Review by siliconivan — APPROVED** [review]
Solid looking guide! LGTM

File: WINDOWS_SETUP.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+# Parabeac-Core setup for Windows
+To get starting with using or developing the source code on Windows some configuration is required. This document is ment to help users with that.
```
Comment (x64Eddie) [inline WINDOWS_SETUP.md]: I think its "ment" instead of "meant" 🤔

File: WINDOWS_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+- [Installing WSL 2](#Installing-WSL-2)
+- [Ubunutu setup](#Ubunutu-setup)
+- [Flutter install](#Flutter-install)
+- [Cloning Parabeac-core](#Cloning-Parabeac-core)
+- [(Optional) Fix NPM](#(Optional)-Fix-NPM)
+- [(Optional) Install VS-Code and WSL 2 extension](#(Optional)-Install-VS-Code-and-WSL-2-extension)
+- [(Optional) Android emulation](#(Optional)-Android-emulation)
+
+### Requirements
+Altough WSL is supported on most Windows installations, there are still some requirements that need to be met before we can continue:
```
Comment (x64Eddie) [inline WINDOWS_SETUP.md]: "Altough" to "Although"? 🤔

File: WINDOWS_SETUP.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+-   For x64:
+    - At least Windows version 1903 with at least build 18362
+
+Altough WSL 2 is supported on ARM64 we'll only focus on x64.
```
Comment (x64Eddie) [inline WINDOWS_SETUP.md]: I think this one should be "Although" as well? 🤔

File: WINDOWS_SETUP.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+### Installing WSL 2
+To summerize the first step, we first need to activate WSL 1 and then update it to version 2. Follow these steps:
```
Comment (x64Eddie) [inline WINDOWS_SETUP.md]: should it be "summarize"? 🤔

File: WINDOWS_SETUP.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+1. Open Windows search and open "Edit the system environment variables"
+
+2. Click on "Environment variables".
+
+3. Select the "Path" variable under user variables.
+
+4. Check if there's a reference to node or npm. Try temporarly removing this.
+
+5. Repeat from step 3, buth then under system variables.
```
Comment (x64Eddie) [inline WINDOWS_SETUP.md]: "buth" -> "but"? 🤔

**Review by x64Eddie — COMMENTED** [review]
Awesome documentation Roel! Other than the small spelling mistakes that I commented out, everything looks good to me. Awesome job!

**Review by x64Eddie — APPROVED** [review]

**Pull request merged**